tests/e2e/pageobjects/dashboard/Dashboard.ts

```diff
@@ -20,7 +20,7 @@ import { Logger } from '../../utils/Logger';
 @injectable()
 export class Dashboard {
     private static readonly WORKSPACES_BUTTON_XPATH: string = `//div[@id='page-sidebar']//a[text()='Workspaces']`;
-    private static readonly GET_STARTED_BUTTON_XPATH: string = `//div[@id='page-sidebar']//a[text()='Get Started Page']`;
+    private static readonly GET_STARTED_BUTTON_XPATH: string = `//div[@id='page-sidebar']//a[text()='Get Started']`;
```

Comment:
@rukshan99 as far as I understand you adapted e2e tests to changes on dashboard which are not done yet.

Would you mind to create another PR against che-dashboard repo,
Get Started Page appears here https://github.com/eclipse/che-dashboard/blob/master/src/Layout/Navigation/MainList.tsx#L27 and in unit tests

Reply:
Dashboard side is fixed by https://github.com/eclipse/che-dashboard/pull/202